InterviewPrep.Angular2FirstLook/wwwroot/app/main.ts

```diff
@@ -1,8 +1,9 @@
-﻿import {bootstrap}    from "angular2/platform/browser";
-import {provide} from "angular2/core"
+﻿import {bootstrap}    from "@angular/platform-browser-dynamic";
+import {provide} from "@angular/core"
 import {AppComponent} from "./appComponent";
-import {ROUTER_PROVIDERS} from "angular2/router"
-import {LocationStrategy, HashLocationStrategy, APP_BASE_HREF, PathLocationStrategy} from "angular2/platform/common"
+import {ROUTER_PROVIDERS} from "@angular/router-deprecated"
+import {LocationStrategy, HashLocationStrategy, APP_BASE_HREF, PathLocationStrategy} from "@angular/common"
+import "rxjs/Rx"
```

Comment:
had to use a [work-around ](url)for a VS 2015 Typescript tooling fix to get RXJS to not throw errors